conf/default.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+reportlab_trusted_hosts: []
```

Comment:
Can you add a comment here that either points to the reportlab documentation on this, or just provide an example? I believe the docs say that glob patterns are allowed for domains, but CIDR blocks aren't likely allowed. Also saying that leaving it empty means that no host restrictions are applied.